src/main/java/com/technocredits/orangeHRM/util/PropertyFileReader.java

```diff
@@ -0,0 +1,31 @@
+package com.technocredits.orangeHRM.util;
+
+import java.io.File;
+import java.io.FileInputStream;
+import java.io.FileNotFoundException;
+import java.io.IOException;
+import java.util.Properties;
+
+public class PropertyFileReader {
+
+	Properties prop=null;
+	public Properties initializePropertyFile(String propFileName) {
+		
+		File file=new File(propFileName+".properties");
+		try {
+			System.out.println("Null pointer");
+			FileInputStream input= new FileInputStream(file);
+			
+			prop=new Properties();
+			
+			prop.load(input);
```

Comment:
Remove extra space in above.
Add comment what this method is actually doing.

Reply:
Extra spaces removed. And comments are added accordingly.